pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java

```diff
@@ -86,14 +109,13 @@ public Multimap<String, String> findBundlesForUnloading(final LoadData loadData,
             double minimumThroughputToOffload = brokerCurrentThroughput * percentOfTrafficToOffload;
 
             if (minimumThroughputToOffload < minThroughputThreshold) {
-                if (log.isDebugEnabled()) {
-                    log.debug("[{}] broker is planning to shed throughput {} MByte/s less than "
+                if (sampleLog) {
```

Comment:
How about use `log.info` directly since `sampleLog` maybe miss some log ?

Reply:
I intentionally sample the logs as this shedding logic computes the load frequently every x min(I think the default is every one minute).